main/src/main/java/org/mobilitydata/gtfsvalidator/validator/UrlConsistencyValidator.java

```diff
@@ -70,72 +59,47 @@
 
   @Override
   public void validate(NoticeContainer noticeContainer) {
-    Map<String, GtfsAgency> agencyByUrlMap = agenciesByUrlMap(agencyTable);
+    ListMultimap<String, GtfsAgency> agencyByUrlMap = agenciesByUrlMap(agencyTable);
+    ListMultimap<String, GtfsRoute> routesByUrlMap = routesByUrlMap(routeTable);
     for (GtfsRoute route : routeTable.getEntities()) {
       if (!route.hasRouteUrl()) {
-        return;
+        continue;
       }
-      GtfsAgency agency = agencyByUrlMap.get(route.routeUrl().toLowerCase());
-      if (agency != null) {
-        noticeContainer.addValidationNotice(
-            new SameRouteAndAgencyUrlNotice(
-                route.csvRowNumber(),
-                route.routeId(),
-                agency.agencyName(),
-                route.routeUrl(),
-                agency.csvRowNumber()));
+      List<GtfsAgency> agencies = agencyByUrlMap.get(route.routeUrl().toLowerCase());
```

Comment:
Here and elsewhere: please use `Ascii.toLowerCase` to make sure that we do not depend on locale. Say, Turkish lowercase for `I` is `ı`, while `i` is a lowercase for `İ`.

```
import com.google.common.base.Ascii;
```

Reply:
This `agencies` variable is used only once, so please inline it. Also below.

Reply:
Thanks for flagging that! Modified in https://github.com/MobilityData/gtfs-validator/pull/1003/commits/4b8eb25762b7ce60ac9d3fea56f28b296318be4d.